Lib/test/test_ssl.py

```diff
@@ -3440,16 +3437,15 @@ def test_do_handshake_enotconn(self):
             self.assertEqual(cm.exception.errno, errno.ENOTCONN)
 
     def test_default_ciphers(self):
```

Comment:
this is called `test_default_ciphers` but the test seems to be explicitly setting which ciphers are used so the name doesn't make much sense. I don't see any defaults.  it appears to be testing that defaults can be overridden and that cipher negotiation fails properly?

Reply:
if i'm misunderstanding, i suggest adding a comment to explain the test. :)